typing/typeclass.ml

```diff
@@ -773,10 +773,12 @@ let rec class_field_first_pass self_loc cl_num sign self_scope acc cf =
                match get_desc ty with
                | Tvar _ ->
                    let ty' = Ctype.newvar () in
-                   Ctype.unify val_env (Ctype.newty (Tpoly (ty', []))) ty;
+                   Ctype.unify val_env
+                      (Ctype.newty (Tpoly {poly_body = ty'; poly_univars = []}))
```

Comment:
Maybe there should be a little function to create a `tpoly` with an emtpy `poly_univars`? Not sure.